invokeai/frontend/web/src/features/lora/components/ParamLoraSelect.tsx

```diff
@@ -59,7 +59,7 @@ const ParamLoRASelect = () => {
       a.label && b.label ? (a.label?.localeCompare(b.label) ? 1 : -1) : -1
     );
 
-    return data.sort((a, b) => (a.disabled && !b.disabled ? -1 : 1));
+    return data.sort((a, b) => (a.disabled && !b.disabled ? 1 : -1));
```

Comment:
This flips the order of the items in the group and sort them from Z-A instead of A-Z.

We need to run another sort for the group name based on which active model is selected in order to make sure the sorted list has the active model items at the top at all times.

Reply:
The model adapters already sort the models alphabetically, there's actually no need to do more sorting - removing the other sort makes it alphabetical

Reply:
Thanks for catching, fixed 
![image](https://github.com/invoke-ai/InvokeAI/assets/4822129/e8095cef-71fd-4786-8142-294f30d1e84d)